examples/notebooks/keras_private_predict.ipynb

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
<p>the call to `get_weights` and `get_config` seem more like an implementation detail, think the right thing to do is *not* to have them but instead call `tfe_model = tfe.keras.models.clone_from(model_k)` (see <a href="https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/models/clone_model" target="_blank">https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/models/clone_model</a>)</p>

 _Reply via <a href='https://app.reviewnb.com/tf-encrypted/tf-encrypted/pull/571/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I agree that's cleaner, but I think we should support both. a user might want to do things like only set some of the weights (common in transfer learning)</p>

Reply:
I like the idea of adding `clone_from`. But let's keep the option to do `set_weights` and `get_config`.

Reply:
sounds good! this is a good target I think: https://www.tensorflow.org/api_docs/python/tf/keras/models